pylace/tests/dummy_test.py

```diff
@@ -0,0 +1,12 @@
+"""Tests the tester, no real test here."""
+
+
+def test_placeholder() -> None:
+    """
+    Test the tester, not expected to fail.
+
+    Tests the tes only exists to ensure that there is at least 1 test for `pytest`,
+    and thus that it does not exit with a failure due to
+    no Python tests existing.
+    """
+    pass
```

Comment:
This probably isn't necessary anymore since `example_test.py` (with `test_animals`) exists, right?